books/RayTracingInOneWeekend.html

```diff
@@ -1044,8 +1044,11 @@
 at geometry time. This is simply a matter of preference, and you'll see both implementations in the
 literature.
 
-We add the `front_face` bool to the `hit_record` class. We'll also add a function to solve this
-calculation for us.
+We add the `front_face` bool to the `hit_record` class.
+We'll also add a function to solve this calculation for us: `set_face_normal()`.
+Note that the vector passed to the new `set_face_normal` function is expected to be of unit length.
+Of course, we could always normalize the parameter explicitly, but it's more efficient if the
+  geometry code does this, as it's usually easier when you know more about the specific geometry.
```

Comment:
"Note that the vector passed to the new `set_face_normal` function is expected to be of unit length. Of course, we could always normalize the parameter explicitly, but it's more efficient if the geometry code does this, as it's usually easier when you know more about the specific geometry."
-> something like
"For convenience we will assume that the vector passed to the new `set_face_normal` function is of unit length. We could always normalize the parameter within the body of the `set_face_normal` function, ...

Reply:
I like it.